build.gradle

```diff
@@ -39,6 +39,18 @@ ext {
     generatedMetamodels = "build/generated/sources/annotationProcessor/java/main/org/candlepin/subscriptions"
 }
 
+spotless {
+    java {
+        target project.fileTree(project.rootDir) {
+            include '**/*.java'
+            exclude '**/build/**'
+        }
+        enforceCheck false
```

Comment:
This line is what allows `./gradlew build` task to be successful, even if there is a code style violation

Reply:
Wanna put that in a comment above the line itself for clarity?